### ISSUE TITLE
HomeConnect API unexpected values (WTX87E40/03)

### ISSUE THREAD

**Issue by smhex** [thoukydides/homebridge-homeconnect#214]
### Plugin Version

0.37.8

### Home Connect Appliance(s)

Bosch Dryer WTX87E40/03

### Log File

```shell
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] ============================================================================================
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] 
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] // Option
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] export interface OptionValues {
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.EnergyForecast'?:                           number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.EstimatedTotalProgramTime'?:                number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.FinishInRelative'?:                         number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.ProgramProgress'?:                          number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.RemainingProgramTime'?:                     number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.RemainingProgramTimeIsEstimated'?:          boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.StartInRelative'?:                          number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'BSH.Common.Option.WaterForecast'?:                            number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'Dishcare.Dishwasher.Option.ExtraDry'?:                        boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'Dishcare.Dishwasher.Option.HygienePlus'?:                     boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'Dishcare.Dishwasher.Option.IntensivZone'?:                    boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'Dishcare.Dishwasher.Option.SilenceOnDemand'?:                 boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'Dishcare.Dishwasher.Option.VarioSpeedPlus'?:                  boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Common.Option.LoadRecommendation'?:               number;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Common.Option.ProcessPhase'?:                     ProcessPhase;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Common.Option.ReferToProgram'?:                   number | string;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.ConnectedDry.OriginalProgramTime'?:  number; // (unrecognised)
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.DryingTarget'?:                      DryingTarget;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.DryingTargetAdjustment'?:            DryingTargetAdjustment;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.Gentle'?:                            boolean;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.WrinkleGuard'?:                      WrinkleGuard;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.SpinSpeed'?:                        SpinSpeed;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect]     'LaundryCare.Washer.Option.Temperature'?:                      Temperature;
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] }
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] 
[1/5/2024, 7:22:48 PM] [homebridge-homeconnect] ============================================================================================
```

### Additional Information

Yet another output from my Bosch Dryer :-) Thanks for keeping this plugin up-to-date...

**Comment by thoukydides** [comment]
Thanks for the report.
The undocumented program option `LaundryCare.Dryer.Option.ConnectedDry.OriginalProgramTime` will be in the next plugin release.

**Comment by thoukydides** [comment]
Should be fixed in [v0.37.9](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v0.37.9).
